go.mod

```diff
@@ -10,4 +10,5 @@ require (
 	github.com/kr/pretty v0.1.0
 	github.com/kr/text v0.1.0 // indirect
 	github.com/vmware/vmw-guestinfo v0.0.0-20170707015358-25eff159a728
+	golang.org/x/tools v0.0.0-20190520220859-26647e34d3c0 // indirect
```

Comment:
Doesn't look like this change is needed?

Reply:
Go just kind of stuffed it in there so I assumed it was needed.

Reply:
go.{mod,sum} should only change if `go mod` commands are used to update dependencies, but possible your IDE had done that automatically?

Reply:
they will also change as needed when running things like `go test`,  and `go build`...

From point 3 in the [wiki](https://github.com/golang/go/wiki/Modules#how-to-define-a-module):
 > go build will automatically add missing or unconverted dependencies as needed to satisfy imports for this particular build invocation

Might worth running `go mod tidy` if you are not already doing so to capture everything.